ts/TreeUtil.ts

```diff
@@ -80,12 +80,13 @@ export class TreeUtil {
     }
 
     async plantAll() {
-        const before = this.railroad.removedVegetationAssets.length;
-        if (before === 0) return;
-        const treesBefore = this.railroad.removedVegetationAssets;
+        // const before = this.railroad.removedVegetationAssets.length;
+        // if (before === 0) return;
+        // const treesBefore = this.railroad.removedVegetationAssets;
         this.railroad.removedVegetationAssets = [];
+        this.railroad.vegetation = []; // temp fix for plant all trees, and fix bugged maps
         this.setMapModified();
-        return this.onTreesChanged(before, 0, treesBefore);
+        // return this.onTreesChanged(before, 0, treesBefore);
```

Comment:
The `onTreesChanged` callback is important - it removes rendered trees from the map.

One approach that might work is to put `this.railroad.vegetation = [];` before the early exit, and avoid changing the cleanup behavior. That might look something like this:
```suggestion
        if (this.railroad.vegetation.length > 0) {
            this.railroad.vegetation = [];
            this.setMapModified();
        }
        const before = this.railroad.removedVegetationAssets.length;
        if (before === 0) return;
        const treesBefore = this.railroad.removedVegetationAssets;
        this.railroad.removedVegetationAssets = [];
        this.setMapModified();
        return this.onTreesChanged(before, 0, treesBefore);
```

However, I am not entirely sure that this is the correct approach. I think it makes more sense to disable the tree tools when `this.vegetation.length > 0`, and instead implement basic "plant all" functionality for the new tree storage format.

Reply:
well its not the correct approach but it would make it workish until someone smarter then me get to it, but imma try the suggestion locally tho and see what happenes

Reply:
#119 adds a tool that should allow you to get out of the broken state. If you want to access the legacy tree utils, you will need to first use the plant all vegetation tool, then export the save and then re-load it. If RS detects that there is no vegetation when the file is loaded, it will enable the legacy tree utils.

Reply:
well good, but how about the radiuses for the smart tree tools and few other small snipets ?